packages/block-editor/src/components/block-list/block-wrapper.js

```diff
@@ -238,13 +240,17 @@ const BlockComponent = forwardRef(
 
 		return blockWrapper;
 	}
-);
 
-const ExtendedBlockComponent = ELEMENTS.reduce( ( acc, element ) => {
-	acc[ element ] = forwardRef( ( props, ref ) => {
-		return <BlockComponent { ...props } ref={ ref } tagName={ element } />;
-	} );
-	return acc;
-}, BlockComponent );
+	return forwardRef( BlockWrapper );
+}
+
+export const block = createHigherOrderComponent(
+	higherOrderComponent,
+	'block'
+);
 
-export const Block = ExtendedBlockComponent;
+ELEMENTS.forEach( ( element ) => {
+	const ElementBlock = block( element );
+	block[ element ] = ElementBlock;
```

Comment:
Should we deprecate the lowercase shortcuts? If so, we should probably add a comment here about it.

Reply:
Since it's experimental, we could just remove it, but I suspect it might be used by plugins by now so I'm hesitant to remove it since it's just one line.

Reply:
Personally, I think it's confusing to support both, so we should at least deprecate the lowercase versions, even if we never remove them.

Reply:
Yes, that's true!